docs/docs/health-checks.md

```diff
@@ -60,6 +60,8 @@ Mesos-level health checks (`MESOS_HTTP`, `MESOS_HTTPS`, `MESOS_TCP`, and `COMMAN
 
 - Mesos-level health checks require tasks to listen on the container's loopback interface in addition to whatever interface they require. If you run a service in production, you will want to make sure that the users can reach it.
 
+- Marathon currently does NOT support the combination of mesos and marathon level health checks.
```

Comment:
nit: `Mesos and Marathon`.

Reply:
I'd prefer to enforce it in `AppDefinition` validation rather than just saying in the documentation

Reply:
hey... @janisz !!!  good point.  I will add a JIRA to that effect!

Reply:
https://jira.mesosphere.com/browse/MARATHON-8369